test/parallel/test-stream-writable-write-writev-finish.js

```diff
@@ -146,7 +146,7 @@ const stream = require('stream');
 
   const ws = new stream.Writable();
 
-  ws.on('finish', common.mustNotCall());
+  ws.on('finish', common.mustCall());
```

Comment:
Even if this is weird (there is a separate issue for 'finish' after 'error')... all the other tests assume exactly this. Only reason this was different was because of the sync/async difference.